frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -8,26 +10,42 @@ import Icon from "metabase/components/Icon";
 import Actions from "metabase/entities/actions";
 import { humanize } from "metabase/lib/formatting";
 
-import type { ModelAction } from "metabase-types/api";
-import type { State } from "metabase-types/store";
-import type Question from "metabase-lib/lib/Question";
+import type { WritebackAction } from "metabase-types/api";
+import type { Dispatch, State } from "metabase-types/store";
 
 import {
   EmptyStateContainer,
   EmptyStateTitle,
 } from "../ModelDetailPage.styled";
 import { ActionListItem, ActionTitle } from "./ModelActionDetails.styled";
+import { hasImplicitActions, isImplicitAction } from "./utils";
+
+const mapDispatchToProps = {
+  enableImplicitActionsForModel: Actions.actions.enableImplicitActionsForModel,
+};
 
 interface Props {
-  model: Question;
-  actions: ModelAction[];
+  modelId: number;
+  actions: WritebackAction[];
+  enableImplicitActionsForModel: (modelId: number) => void;
 }
 
-function ModelActionDetails({ actions }: Props) {
+function ModelActionDetails({
+  actions,
+  modelId,
+  enableImplicitActionsForModel,
+}: Props) {
+  const createImplicitActions = async () => {
+    await enableImplicitActionsForModel(modelId);
+  };
+
   if (!actions?.length) {
     return (
       <EmptyStateContainer>
         <EmptyStateTitle>{t`This model does not have any actions yet.`}</EmptyStateTitle>
+        <Button onClick={createImplicitActions} icon="add">
+          {t`Enable implicit actions`}
+        </Button>
         <Button
           as={Link}
           to={`/action/create`}
```

Comment:
nit (not really related to this PR):

1. Usually it's preferred not to use backticks when you don't need a template string (it can be confusing)
2. For strings I guess it's preferred to use a React string prop shortcut like `to="/action/create"`

Reply:
🤔  this feels like a job for the linter. fixed it here manually.